### DISCUSSION TITLE
Celebrities Images Generation

### DISCUSSION THREAD

**Discussion by georgestefanz** [lllyasviel/Fooocus#3638] [Q&A]
Hi guys, ik it may be a newbie question.
For the last few days i've been trying to figure out how to improve and have the best outcome possible when creating images with famous celebrities. 
For example, when i ask the ai to create donald trump, the outcome is very good. I don't have to work too much on the prompt to get a good result (good result meaning his face, being appropiate to the reality)
When i use a similar prompt to recreate Cristiano Ronaldo for example, his face looks very unfamiliar compared to "the real life version".
So, how can i have the best outcome when creating celebrities? Didn't find anything on youtube about this on foocus.
Thanks !

**Comment by Tomcreating** [comment]
It is hit or miss without specialized Loras. Check civitai site for Loras that are designed on that or other celebrities.

**Reply by georgestefanz** [reply]
Thank you so much for your answer

**Comment by Yogesh-DevHub** [comment]
The Donald Trump example you gave works because the model you are using had Donald Trump's image while training. Now what you can do is try searching for lora of the celebrity . 
Or make a lora by 15 - 30 images of the celebrity. I made one of celebrity Alexandra in the video. 
https://youtu.be/-L9tP7_9ejI?si=L4KZriEKH4M7jzPx

**Reply by georgestefanz** [reply]
Much appreciated, thanks !
